turbo/stages/mock_sentry.go

```diff
@@ -193,7 +193,7 @@ func MockWithEverything(t *testing.T, gspec *core.Genesis, key *ecdsa.PrivateKey
 	} else {
 		tmpdir = os.TempDir()
 	}
-	dirs := datadir.New(tmpdir)
+	dirs := datadir.New(tmpdir, "")
```

Comment:
probably also need pass `tmpdir` as second parameter here

Reply:
> probably also need pass `tmpdir` as second parameter here - otherwise tests may create files in workdir

That's a good idea. I'll also make sure all tests pass before submitting a new pull request. I've tried make test and make test-integration locally... is there another test I need to do? I don't think the consensus tests are relevant, so I assume you mostly mean the unit tests (*_test.go)?

Reply:
I just didn’t know that using empty string here - means use first arg as parent dir